### PR TITLE
Chef 14

### PR DESCRIPTION
**Pull request by Vancelot11**

### CODE REVIEW

**Comment by ramereth** [comment]
@Vancelot11 can you please rebase and fix the conflict?

File: metadata.rb [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
 supports         'centos', '~> 7.0'
 
-depends          'build-essential'
```
Comment (ramereth) [inline metadata.rb]: This can't be removed until we fully migrate to 14. Please keep.

File: recipes/default.rb [hunk 2]
```diff
@@ -57,8 +57,9 @@
 temp = resources(template: '/etc/sudoers')
 temp.variables['include_sudoers_d'] = true
 
+build_essential 'install tools'
```
Comment (ramereth) [inline recipes/default.rb]: Let's keep this in the same order as we have for the recipe inclusion.

**Comment by Vancelot11** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.3.0.  Have a nice day!

**Pull request merged**